pkgs/by-name/zs/zsv/package.nix

```
@@ -8,13 +8,13 @@
 
 stdenv.mkDerivation rec {
```

while you're refactoring switching to finalAttrs would be good
```suggestion
stdenv.mkDerivation (finalAttrs: {
```